### ISSUE TITLE
let's try to bring this to html widgets 

### ISSUE THREAD

**Issue by abresler** [timelyportfolio/rCharts_addepar_ember#1]

**Comment by timelyportfolio** [comment]
I agree it would be a nice widget.  I hesitate though since it is hard to hide the Ember dependencies and requisite knowledge from the `R` user.  I'll try to think through it again though.

**Comment by abresler** [comment]
Ahhh hmm can bower help with that, maybe Ramnath  has some ideas?

---

Alex Bresler
abresler@asbcllc.com
​www.asbcllc.com​
917-455-0239​ (cell)​

**Comment by timelyportfolio** [comment]
I'm not sure I worded my first response in the best way.  Creating the widget is sort-of-trivial.  Making the widget so that a user does not need to understand Ember is much more difficult.  So far, I have tried to focus on widgets that can be used out of the box by an R user who has no knowledge of JavaScript.  This is quite a challenge, so taking the next step to also hide the Ember complexities is just another hurdle.  I'm sure thought with some careful thought and quite a bit of iteration we could eventually accomplish this.  Let me know if you have thoughts.

**Comment by abresler** [comment]
That's fantastic feedback, just getting to ember and didn't realize the
complexity it highlights the splits in the JS  universe of easy to
implement widgets versus frameworks.

I'll look into it more, I'm exploring ember

---

Alex Bresler
abresler@asbcllc.com
​www.asbcllc.com​
917-455-0239​ (cell)​

**Comment by timelyportfolio** [comment]
The nicest features of this is the ability to render sparklines and other graphical element in the table.  This would be particularly helpful for something like a forest plot, which I have started to explore.

I had wondered if ember and jquery would cause conflicts in a Shiny context, but I remembered that they actually use jquery-ui components in their implementation, so that shouldn't be an issue.

**Comment by abresler** [comment]
Also while I like pivotjs  a datatable I think ember tables are 100x better

---

Alex Bresler
abresler@asbcllc.com
​www.asbcllc.com​
917-455-0239​ (cell)​
